### PR TITLE
Skip delete when id doesn’t exists

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue https://github.com/joomla/joomla-cms/commit/c0eadd9f9f31be59f7088b25f66aea346a8e42c3

### Summary of Changes

Just spaces to tabs + two additional inline comments

### Testing Instructions

- Just review
- or install the patch and run the database fixer
. confirm it still works.

### Expected result

match the code standards

### Actual result

does not match the code standards

### Documentation Changes Required

None

### CODE REVIEW

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -344,10 +345,11 @@ protected function removeJedUpdateserver()
 					->where($db->quoteName('location') . ' = ' . $db->quote('https://update.joomla.org/jed/list.xml'))
 			)->loadResult();
 
-            if (!$id)
-            {
-                return;
-            }
+			// Skip delete when id doesn’t exists
+			f (!$id)
+			{
```
Comment (dgt41) [inline administrator/components/com_admin/script.php]: you are missing the `i` in the `if`

Comment (zero-24) [inline administrator/components/com_admin/script.php]: fixed

**Review by richard67 — APPROVED** [review]
I can confirm it only changes code style and adds useful comments.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 270e8caebbf49d2cda2d88b72bd01b2ed37749aa<br /><br />Tested by code review. I can confirm it only changes code style and adds useful comments.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14247">issues.joomla.org/tracker/joomla-cms/14247</a>.</sub>

**Comment by richard67** [comment]
P.S.: And of course I can also confrirm it matches code standards now.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14247">issues.joomla.org/tracker/joomla-cms/14247</a>.</sub>

**Comment by Sieger66** [comment]
I have tested this item :white_check_mark: successfully on 270e8caebbf49d2cda2d88b72bd01b2ed37749aa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14247">issues.joomla.org/tracker/joomla-cms/14247</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC as there are 2 successfully Tests?

**Pull request merged**